packages/MHDDoS/PKGBUILD

```diff
@@ -0,0 +1,43 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+
+pkgname=MHDDoS
+pkgver=2.4.1
+pkgrel=1
+pkgdesc='Best DDoS Attack Script Python3, (Cyber / DDos) Attack With 56 Methods.'
```

Comment:
description shouldn't be promotional eg. using _best_ but rather neutral

Reply:
i copied exactly fro the discription  becoz tool owner is not mine  but i will keep it mind